cas-server-support-ldap/src/test/java/org/jasig/cas/authentication/principal/PersonDirectoryPrincipalResolverLdaptiveTests.java

```diff
@@ -0,0 +1,48 @@
+package org.jasig.cas.authentication.principal;
+
+import org.jasig.cas.adaptors.ldap.AbstractLdapTests;
+import org.jasig.cas.authentication.UsernamePasswordCredential;
+import org.jasig.services.persondir.support.ldap.LdaptivePersonAttributeDao;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.ldaptive.LdapEntry;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import static org.junit.Assert.*;
+
+/**
+ * This is {@link PersonDirectoryPrincipalResolverLdaptiveTests}.
+ *
+ * @author Misagh Moayyed
+ * @since 4.3.0
+ */
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration({"/ldap-context.xml", "/resolver-context.xml"})
+public class PersonDirectoryPrincipalResolverLdaptiveTests extends AbstractLdapTests {
```

Comment:
How is this related to the pull request? And it's not the first change which seems unrelated: any sync issue?